tests/Sdk/Unit/Presentation/Console/Manifest/Task/ValueResolver/ValuesResolverMapRegistryTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Sdk\Unit\Presentation\Console\Manifest\Task\ValueResolver;
+
+use Codeception\Test\Unit;
+use InvalidArgumentException;
+use SprykerSdk\Sdk\Core\Application\Registry\RegistryItemInterface;
+use SprykerSdk\Sdk\Presentation\Console\Manifest\Task\ValueResolver\ValuesResolverMapRegistry;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group Unit
+ * @group Presentation
+ * @group Console
+ * @group Manifest
+ * @group Task
+ * @group ValueResolver
+ * @group ValuesResolverMapRegistryTest
+ * Add your own group annotations below this line
+ */
+class ValuesResolverMapRegistryTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testHasShouldCheckExistenceOfRegistryItem(): void
+    {
+        // Arrange
+        $item = $this->createRegistryItem('one');
+        $registry = new ValuesResolverMapRegistry([$item]);
+
+        // Act
+        $itemOneStatus = $registry->has('one');
+        $itemTwoStatus = $registry->has('two');
+
+        // Assert
+        $this->assertTrue($itemOneStatus);
```

Comment:
Please, provide an additional message if the test method has more than one assert check. The same for the `testFindShouldReturnItemByName`